client/verifier.go

```diff
@@ -68,14 +72,14 @@ func (c *logVerifier) VerifyRoot(trusted, newRoot *trillian.SignedLogRoot,
 // the currently trusted root. The inclusion proof must be requested for Root().TreeSize.
 func (c *logVerifier) VerifyInclusionAtIndex(trusted *trillian.SignedLogRoot, data []byte, leafIndex int64, proof [][]byte) error {
 	leaf := c.buildLeaf(data)
-	return c.v.VerifyInclusionProof(leafIndex, trusted.TreeSize,
-		proof, trusted.RootHash, leaf.MerkleLeafHash)
+	return c.v.VerifyInclusionProof(leafIndex, trusted.GetTreeSize(),
```

Comment:
A panic (SEGFAULT) is bad for gobind integration (the android app hangs) and might cause errors too, so I would go for an error instead.